src/routes/search.route.js

```diff
@@ -0,0 +1,42 @@
+import { Router } from 'express';
+import tripController from '../controllers/trip.controller';
+import verifyToken from '../middlewares/verify.token.middleware';
+
+const router = Router();
+
+/**
+ * @swagger
+ * /trip-requests/search:
+ *    get:
+ *      summary: user and manager can search for a trip request
+ *      tags: [search]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: keyword
+ *         in: query
+ *         description: query to search for
+ *         required: true
+ *         type: string
+ *       - name: page
+ *         in: query
+ *         schema:
+ *           type: integer
+ *         description: page number
+ *         required: true
+ *       - name: limit
+ *         in: query
+ *         schema:
+ *           type: integer
+ *         description: limit number
+ *         required: true
+ *      responses:
+ *        "200":
+ *          description: Search record schema
+ */
+router.get('/search', verifyToken.headerToken, tripController.searchAtripRequest);
```

Comment:
Good work so far @jo-rdan but I was confused on what you are searching for,if it is trip request only or if it is a certain keyword, based on this function it implies that you only searching a trip request only.